src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -175,10 +175,18 @@ public void Version_MatchesFixedVersion()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // Throws InvalidOperationException in Uap as NtQuerySystemInformation Pinvoke is not available
         public void WorkingSet_Valid()
         {
             Assert.True(Environment.WorkingSet > 0, "Expected positive WorkingSet value");
         }
+        
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] 
+        public void WorkingSet_Valid_Uap()
+        {
+            Assert.Throws<InvalidOperationException>(() => Environment.WorkingSet);
```

Comment:
That's right. The only reason I did it this way was hoping that some scenarios would work. But I guess accessing the process counter is not allowed in Uap anyway (not 100% sure though). Considering we will eventually fix this, I agree throwing PNSE is less confusing. Changed to that.